### DISCUSSION TITLE
Dealing with multi-threaded execution issues

### DISCUSSION THREAD

**Discussion by wondering639** [appany/AppAny.HotChocolate.FluentValidation#89] [Q&A] [answered]
I am just starting with HotChocolate/GraphQL and looking for a validation library. Both, FairyBread and [AppAny.HotChocolate.FluentValidation](https://github.com/appany/AppAny.HotChocolate.FluentValidation) look very interesting to me. On the FairBread readme I've noted a hint about [Dealing with multi-threaded execution issues](https://github.com/benmccallum/fairybread#dealing-with-multi-threaded-execution-issues) and how they offer a solution for this. Which solution does [AppAny.HotChocolate.FluentValidation](https://github.com/appany/AppAny.HotChocolate.FluentValidation)  provide for this or how would I solve this manually? Thanks!

**Comment by sergeyshaykhullin** [comment, marked as answer]
This library rely on DI scopes, there is no such problems
